pkg/scheduler/framework/runtime/framework.go

```diff
@@ -252,25 +265,27 @@ func NewFramework(r Registry, profile *config.KubeSchedulerProfile, stopCh <-cha
 	}
 
 	f := &frameworkImpl{
-		registry:             r,
-		snapshotSharedLister: options.snapshotSharedLister,
-		scorePluginWeight:    make(map[string]int),
-		waitingPods:          newWaitingPodsMap(),
-		clientSet:            options.clientSet,
-		kubeConfig:           options.kubeConfig,
-		eventRecorder:        options.eventRecorder,
-		informerFactory:      options.informerFactory,
-		metricsRecorder:      options.metricsRecorder,
-		extenders:            options.extenders,
-		PodNominator:         options.podNominator,
-		parallelizer:         options.parallelizer,
+		registry:                          r,
+		snapshotSharedLister:              options.snapshotSharedLister,
+		scorePluginWeight:                 make(map[string]int),
+		waitingPods:                       newWaitingPodsMap(),
+		clientSet:                         options.clientSet,
+		kubeConfig:                        options.kubeConfig,
+		eventRecorder:                     options.eventRecorder,
+		informerFactory:                   options.informerFactory,
+		metricsRecorder:                   options.metricsRecorder,
+		extenders:                         options.extenders,
+		PodNominator:                      options.podNominator,
+		parallelizer:                      options.parallelizer,
+		schedulerPercentageOfNodesToScore: options.schedulerPercentageOfNodesToScore,
```

Comment:
OK, reverted `numFeasibleNodesToFind` to a scheduler's member func so it can obtain the global level value.  @Huang-Wei , can you take a look too? Thanks.

```
// numFeasibleNodesToFind returns the number of feasible nodes that once found, the scheduler stops
// its search for more feasible nodes.
func (sched *Scheduler) numFeasibleNodesToFind(percentageOfNodesToScore *int32, numAllNodes int32) (numNodes int32) {
    if numAllNodes < minFeasibleNodesToFind {
        return numAllNodes
    }

    // Use the profile percentageOfNodesToScore it it's not nil. If the profile level percentageOfNodesToScore
    // is nil and the global percentageOfNodesToScore is not nil, use the global percentageOfNodesToScore.
    // Otherwise, use the default percentageOfNodesToScore.
    var percentage int32
    percentage = int32(config.DefaultPercentageOfNodesToScore)
    if percentageOfNodesToScore != nil {
        percentage = *percentageOfNodesToScore
    } else {
        percentage = sched.percentageOfNodesToScore
    }
```